caffe2/python/pybind_state_gpu.cc

```diff
@@ -66,6 +67,17 @@ void addCUDAGlobalMethods(py::module& m) {
     obj["totalGlobalMem"] = py::cast(prop.totalGlobalMem);
     return obj;
   });
+  m.def(
```

Comment:
it won't compile if we compile without USE_TENSORRT, right? we probably should create a tiny separate pybind

Reply:
Yes. Plumbing through the flow now. Will change.